core-bundle/src/DependencyInjection/Compiler/MakeServicesPublicPass.php

```diff
@@ -33,8 +33,8 @@ class MakeServicesPublicPass implements CompilerPassInterface
         'security.authentication.trust_resolver',
         'security.firewall.map',
         'security.helper',
-        'security.logout_url_generator',
         'security.encoder_factory',
+        'security.logout_url_generator',
```

Comment:
Would the service correctly sorted not sit after `security.authentication.trust_resolver`?

Reply:
Like in f0e377785?

Reply:
Yes. 🙈